### ISSUE TITLE
oh-my-zsh script does not exist.

### ISSUE THREAD

**Issue by marcdw1289** [adi1090x/termux-omz#1]
I setup termux-omz and all looked good. After exiting and restarting Termux I'm presented with a zsh error regarding this line...

source $ZSH/.oh-my-zsh.sh

That script exists nowhere under ZSH or HOME. Is that something left over from the past or is there a bug in the setup? My terminal looks okay but not quite like the screenshots so maybe something's missing(?).

**Comment by adi1090x** [comment]
Seems like a path issue, try to install on a fresh termux installation.

**Comment by marcdw1289** [comment]
Thanks. On another ROM I hit the same snag as I had started off with the Termux Beautiful Desktop thing. Oh-my-zsh was listed near the end of the README as an additional add-on so I went to that after. That's when I caught an error during its setup that I didn't notice on the other device.

So I started clean again. Oh-my-zsh first, Termux Style scripts, then Beautiful Desktop afterward. All good with the exception of the scripts trying to load some spaceship theme that doesn't exist in the directory of themes.
Thanks again.
